### PR TITLE
Pivotal

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

**Comment by angy113** [comment]
@carledriss pivotal project

**Comment by angy113** [comment]
@carledriss Shapes ready to code review

File: src/main/java/Main.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+public class Main {
```
Comment (carledriss) [inline src/main/java/Main.java:2]: Remove this unnecessary class

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**